metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/utils/EnrichmentFields.java

```diff
@@ -1,4 +1,4 @@
-package eu.europeana.metis.utils;
+package eu.europeana.enrichment.utils;
 
 import eu.europeana.corelib.definitions.jibx.Contributor;
 import eu.europeana.corelib.definitions.jibx.Coverage;
```

Comment:
This class could probably do with some JavaDoc.

Reply:
This entire class needs a massive rewrite/refactor. This can best be done in upcoming sprints after we come up with a larger cleanup strategy together.